smoke-tests/framework/src/main/java/com/microsoft/applicationinsights/smoketest/fakeingestion/MockedQuickPulseServlet.java

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.applicationinsights.smoketest.fakeingestion;
+
+import com.google.common.io.CharStreams;
+import java.io.IOException;
+import java.io.StringWriter;
+import java.rmi.ServerError;
```

Comment:
```suggestion
```

The test is not related to RMI.

An `IllegalStateException` would be fine for me.